AutomatedTesting/Gem/PythonTests/scripting/ScriptCanvasComponent_OnEntityActivatedDeactivated_PrintMessage.py

```diff
@@ -134,55 +130,51 @@ def validate_start_state(entity: EditorEntity, expected_state: str):
             entity.set_start_status(expected_state)
         return True
```

Comment:
This will always return true as written which means that any of the logic used from this function will always be true if it exits. Otherwise we're raising errors. This function likely needs to be re-thought out.

Reply:
Additionally, this function should be a little more meaningful `validate_entity_state` or `validate_entity_start_state`.

Lastly, is this function also another scripting_tools function candidate?

Reply:
I migrated this to the tools file and updated the return statement to check if the expected state is equal to the actual state. We  need to have a quick chat about the 'state_options' variable though. Originally, it was drawing from a global variables file. I spent some time trying to get the global variables to work with the script tools file but it just wouldn't. I'm not sure if the solution I have here is safe,